next-docs/pages/cli/worldgen.mdx

```diff
@@ -1 +1,30 @@
 # mud worldgen
+
+Autogenerate MUD Store table libraries based on the config file
+
+## Sample usage
+
+Use the `System`s under `src`<sup>1</sup> to generate the Solidity interface definitions for those `Systems` and put them in `src/codegen/world`.
+Also, create `src/codegen/world/IWorld.sol` which imports all the `System` interfaces.
```

Comment:
should we add a sample cli command here that people can copy?
probably just
```
pnpm mud worldgen
```

Reply:
Yes, sorry I forgot - too much working while tired.